dbt/adapters/bigquery/connections.py

```diff
@@ -449,19 +451,22 @@ def execute(
             num_rows = query_table.num_rows
             num_rows_formated = self.format_rows_number(num_rows)
             bytes_processed = query_job.total_bytes_processed
+            slot_ms = query_job.slot_ms
```

Comment:
Thank you @yu-iskw for working towards adding this deeper ability to control usage. While testing it locally I happened to notice that we would get an error for `'QueryJob' object has no attribute 'slot_ms'` due to the name for slot milliseconds in [Class Query Job](https://cloud.google.com/python/docs/reference/bigquery/latest/google.cloud.bigquery.job.QueryJob) is actually `slots_millis` renaming in 4 locations in the file like below should get tests passing more cleanly.

```suggestion
            slot_ms = query_job.slot_millis
```